### ISSUE TITLE
No matching export in "node_modules/@astrojs/db/dist/runtime/index.js" for import "sql"

### ISSUE THREAD

**Issue by hkbertoson** [pilcrowOnPaper/lucia-adapter-astrodb#1]
```
node_modules/lucia-adapter-astrodb/dist/index.js:1:9:
      1 │ import { sql } from "@astrojs/db/runtime";
```

Seems like this is not exported from `@astrjojs/db/runtime`

**Comment by xrexy** [comment]
Seems like [this commit](https://github.com/withastro/astro/commit/219c49473fe44d8df2b69444b2dce0f5bc971655#diff-fc194e4251e09df951cd32b0045a078e7adba04dff880bd773741eb8be088adaL15) removes exports for both `sql` and `SqliteDB` which are imports from drizzle-orm, so a possible fix could be importing from drizzle-orm not @astrojs/db/runtime. 

Not sure if it should be fixed in @astrojs/db(or why they even changed it) or if pilcrowOnPaper is fine with the "fix" I mentioned above

**Comment by xrexy** [comment]
Nvm, I'm dumb [this PR](https://github.com/withastro/astro/pull/10457) moves them to /runtime/config.ts

Edit: they have this in their package.json `"./dist/runtime/config.js": { "import": "./dist/runtime/config.js" }` so not sure if you can even import the config lol

**Comment by hkbertoson** [comment]
Yeah im not sure on this. Quite odd

**Comment by pilcrowonpaper** [comment]
I'm not really sure what API to use since `eq()` etc exported by Drizzle and Astro DB have different types. Drizzle isn't 1.0 yet, so I'd prefer to stick to whatever version Astro DB is using too.

**Comment by hkbertoson** [comment]
Sounds like something to talk to the Astro team about.

**Comment by theDeal** [comment]
What is a solution you guys are using rn?

**Comment by xrexy** [comment]
I just went back to @astrojs/db 0.8.6, but wouldn't say that's a solution

**Comment by josu-dev** [comment]
> I just went back to @astrojs/db 0.8.6, but wouldn't say that's a solution

I dont know if its a possible solution but i locally changed my dist file of the adapter to `import {sql} form 'astro:db'` direclty but it breaks with the bundler i think

**Comment by boycott** [comment]
Whats worked for me is to copy the adapter into my lib directory and call it from there, and adapt it as follows:

```ts
import { sql } from 'astro:db';

import type { db as SqliteDB } from 'astro:db';
import type { Table } from "@astrojs/db/runtime";
```
and then use typeof in the private variable and constructor:

```ts
private db: typeof SqliteDB;
...
constructor(db: typeof SqliteDB, sessionTable: SessionTable, userTable: UserTable) {
...
}
```

**Comment by pilcrowonpaper** [comment]
Fixed with 0.0.5 - see the README for configuring your project
